src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule712paragraphs/InputIncorrectJavadocParagraph.java

```diff
@@ -128,14 +128,14 @@ boolean emulated() {
         }
       };
 
-  /* 4 lines below, no violation until #15503 */
+  /* 4 lines below, no violation until #15685 */
```

Comment:
had to change issue number to https://github.com/checkstyle/checkstyle/issues/15685 because CI was complaining. These false-negatives will be covered as part of newly linked issue as they have their corresponding closing tag